ci/github/phpunit.oci8.xml

```diff
@@ -33,10 +33,4 @@
             <directory suffix=".php">../../lib</directory>
         </whitelist>
     </filter>
-
-    <groups>
-        <exclude>
-            <group>performance</group>
-        </exclude>
-    </groups>
```

Comment:
Also, where does this come from?

Reply:
See https://github.com/doctrine/dbal/pull/3923.